File: Week-Two/compareTriplets.py
def compareTriplets(a, b):
    # Write your code here 
    output = [0, 0]
    l = len(a) 

    for x in range(l): 
        if a[x] > b[x]: 
            output[0] += 1 
        elif a[x] < b[x]: 
            output[1] += 1 
        else: 
            continue
            
    print(output) 


a = [17, 28, 30] 
b = [99, 16, 8] 

compareTriplets(a,b)

File: Week-Two/compareTriplets.js
function compareTriplets(a, b) {
    // Write your code here
    let sumA = 0; 
    let sumB = 0; 
    let output = [];  
    
    for (let i=0; i < a.length; i++ ){
        if (a[i] > b[i]){
           sumA += 1;
        } else if(b[i] > a[i]){
           sumB += 1; 
        } 
        output = [sumA, sumB]; 
    } 
    
    return output; 
}

let a = [17, 28, 30]; 
let b = [99, 16, 8]; 

compareTriplets(a,b); 